### DISCUSSION TITLE
How "Long texts example" very fast?

### DISCUSSION THREAD

**Discussion by karthikcodes6** [ueberdosis/tiptap#2756] [General Questions]
I've been using many text editors slate, lexical, etc. I know tiptap internally using prosemirror. But when I insert the long text in prosemirror the whole page hangs. But the same working very well in tiptap. 

I just wanted to know, did you guys make any hack to work fine in tiptap. I have not dig inside the tiptap code base yet. Can anyone tell me how tiptap is optimised internally? Is there any HTML/CSS hack or Reconciliation process? Just curious.

**Comment by sereneinserenade** [comment]
I think the long text example is very fast because of `spellcheck="false"` if you turn it on, it hangs big time fore me

**Reply by aehlke** [reply]
possible to split up spellcheck=true for just nodes in-view dynamically?
